### PR TITLE
feat: added invite team member from onboarding flow

### PR DESCRIPTION
**Pull request by SagarRajput-7**
#### Screenshots

https://github.com/SigNoz/signoz/assets/162284829/bd1653f3-7c6f-4118-99cb-40c554d395ea

Screenshot 2024-07-02 at 8 05 09 PM
Screenshot 2024-07-02 at 8 05 57 PM

### CODE REVIEW

File: frontend/jest.config.ts [hunk 1]
```diff
@@ -23,7 +24,7 @@ const config: Config.InitialOptions = {
 		'^.+\\.(js|jsx)$': 'babel-jest',
 	},
 	transformIgnorePatterns: [
-		'node_modules/(?!(lodash-es|react-dnd|core-dnd|@react-dnd|dnd-core|react-dnd-html5-backend|axios|@signozhq/design-tokens|d3-interpolate|d3-color)/)',
+		'node_modules/(?!(lodash-es|react-dnd|core-dnd|@react-dnd|dnd-core|react-dnd-html5-backend|axios|@signozhq/design-tokens|d3-interpolate|d3-color|react-markdown|vfile|vfile-message)/)',
```
Comment (vikrantgupta25) [inline frontend/jest.config.ts:27]: are these still required?

File: frontend/src/container/OrganizationSettings/InviteUserModal/InviteUserModal.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+					},
+				);
+
+				setTimeout(async () => {
+					const { data, status } = await getPendingInvitesResponse.refetch();
+					if (status === 'success' && data.payload) {
+						setDataSource?.(getParsedInviteData(data?.payload || []));
+					}
+					setIsInvitingMembers?.(false);
+					toggleModal?.(false);
```
Comment (vikrantgupta25) [inline frontend/src/container/OrganizationSettings/InviteUserModal/InviteUserModal.tsx:125]: remove the conditional check

**Review by vikrantgupta25 — DISMISSED** [review]

File: frontend/src/container/OrganizationSettings/InviteUserModal/InviteUserModal.tsx (hunk 2 above)
Comment (SagarRajput-7) [reply frontend/src/container/OrganizationSettings/InviteUserModal/InviteUserModal.tsx]: Done

File: frontend/jest.config.ts (hunk 1 above)
Comment (SagarRajput-7) [reply frontend/jest.config.ts]: Removed

File: frontend/public/locales/en-GB/onboarding.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (YounixM) [inline frontend/public/locales/en-GB/onboarding.json:1]: Create a task to get rid of en-GB files as most of the content looks same.

File: frontend/src/container/OnboardingContainer/OnboardingContainer.tsx [hunk 4]
```diff
@@ -279,58 +285,110 @@ export default function Onboarding(): JSX.Element {
...
-											{selectedUseCase.desc}
-										</Typography.Paragraph>
-									</Card>
-								);
-							})}
+											<Typography.Title
+												className="moduleTitleStyle"
+												level={4}
+												style={{
+													borderBottom: isDarkMode ? '1px solid #303030' : '1px solid #ddd',
```
Comment (YounixM) [inline frontend/src/container/OnboardingContainer/OnboardingContainer.tsx:343]: Get rid of inline styles.

File: frontend/src/container/OrganizationSettings/InviteUserModal/InviteUserModal.tsx [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+		getPendingInvitesResponse?.data?.payload,
+		getPendingInvitesResponse.status,
+		setDataSource,
+	]);
+
+	const onInviteClickHandler = useCallback(
+		async (values: InviteMemberFormValues): Promise<void> => {
+			try {
+				setIsInvitingMembers?.(true);
+				values.members.forEach(
```
Comment (YounixM) [inline frontend/src/container/OrganizationSettings/InviteUserModal/InviteUserModal.tsx:94]: null check

File: frontend/src/container/OnboardingContainer/OnboardingContainer.tsx (hunk 4 above)
Comment (SagarRajput-7) [reply frontend/src/container/OnboardingContainer/OnboardingContainer.tsx:343]: These were existing codes, just added a wrapper div, hence showing as dif, but will check

File: frontend/src/container/OnboardingContainer/OnboardingContainer.tsx [hunk 6]
```diff
@@ -279,58 +285,110 @@ export default function Onboarding(): JSX.Element {
...
-											backgroundColor: isDarkMode ? '#000' : '#FFF',
-										}}
-										key={selectedUseCase.id}
-										onClick={(): void => handleModuleSelect(selectedUseCase)}
-									>
-										<Typography.Title
-											className="moduleTitleStyle"
-											level={4}
+				<div className="onboarding-page">
+					<div>
```
Comment (YounixM) [inline frontend/src/container/OnboardingContainer/OnboardingContainer.tsx:308]: unnecessary element

File: frontend/public/locales/en-GB/onboarding.json (hunk 3 above)
Comment (SagarRajput-7) [reply frontend/public/locales/en-GB/onboarding.json:1]: https://github.com/SigNoz/signoz/issues/5446

File: frontend/src/container/OnboardingContainer/OnboardingContainer.tsx (hunk 6 above)
Comment (SagarRajput-7) [reply frontend/src/container/OnboardingContainer/OnboardingContainer.tsx:308]: removed this.

File: frontend/src/container/OnboardingContainer/OnboardingContainer.tsx (hunk 4 above)
Comment (SagarRajput-7) [reply frontend/src/container/OnboardingContainer/OnboardingContainer.tsx:343]: done

File: frontend/jest.config.ts [hunk 7]
```diff
@@ -9,6 +9,7 @@ const config: Config.InitialOptions = {
 	modulePathIgnorePatterns: ['dist'],
 	moduleNameMapper: {
 		'\\.(css|less|scss)$': '<rootDir>/__mocks__/cssMock.ts',
+		'\\.md$': '<rootDir>/__mocks__/cssMock.ts',
```
Comment (vikrantgupta25) [inline frontend/jest.config.ts:12]: NIT: rename the mock file from `cssMock.ts`

File: frontend/src/container/OnboardingContainer/Onboarding.styles.scss [hunk 8]
```diff
@@ -61,6 +51,8 @@
 	width: 300px;
 	transition: 0.3s;
 
+	background-color: #000;
```
Comment (vikrantgupta25) [inline frontend/src/container/OnboardingContainer/Onboarding.styles.scss:54]: NIT: use css variables

File: frontend/src/container/OnboardingContainer/Onboarding.styles.scss [hunk 9]
```diff
@@ -80,6 +72,9 @@
 	overflow: hidden;
 	text-overflow: ellipsis;
 	text-align: center;
+
+	border-bottom: 1px solid #303030;
```
Comment (vikrantgupta25) [inline frontend/src/container/OnboardingContainer/Onboarding.styles.scss:76]: here as well if present in design system

**Review by vikrantgupta25 — APPROVED** [review]
NIT comments, can be picked up later as well. LGTM

File: frontend/src/container/OnboardingContainer/Onboarding.styles.scss (hunk 8 above)
Comment (SagarRajput-7) [reply frontend/src/container/OnboardingContainer/Onboarding.styles.scss:54]: didn't found any matches

File: frontend/src/container/OnboardingContainer/Onboarding.styles.scss (hunk 9 above)
Comment (SagarRajput-7) [reply frontend/src/container/OnboardingContainer/Onboarding.styles.scss:76]: didn't found any matches

File: frontend/jest.config.ts (hunk 7 above)
Comment (SagarRajput-7) [reply frontend/jest.config.ts:12]: they should be referring to same file content, hence didn't created a duplicate, will do in next PR

Comment (vikrantgupta25) [reply frontend/jest.config.ts:12]: yes so update it to a generic name. not super important but better to maintain code quality

**Review by YounixM — APPROVED** [review]

**Pull request merged**